### PR TITLE
doc: add config help/get/set section for runtime client configuration

### PR DESCRIPTION
**Pull request by vshankar**
Fixes: http://tracker.ceph.com/issues/41688
Signed-off-by: Venky Shankar <vshankar@redhat.com>

### CODE REVIEW

**Comment by vshankar** [comment]
jenkins render docs

**Comment by vshankar** [comment]
jenkins render docs

**Comment by vshankar** [comment]
jenkins render docs

File: doc/cephfs/client-config-ref.rst [hunk 1]
```diff
@@ -1,6 +1,48 @@
...
+
+   e.g.,
+   ceph config help debug_client
+    debug_client - Debug level for client
+    (str, advanced)                                                                                                                      Default: 0/5
+    Can update at runtime: true
+
+    The value takes the form 'N' or 'N/M' where N and M are values between 0 and 99.  N is the debug level to log (all values below this are included), and M is the level to gather and buffer in memory.  In the event of a crash, the most recent items <= M are dumped to the log file.
+
+`config help` tells is the given configuration can be applied at runtime along with the defaults and a description of the configuration option.
```
Comment (batrick) [inline doc/cephfs/client-config-ref.rst]: `s/tells is/tells us/`?

Comment (vshankar) [reply doc/cephfs/client-config-ref.rst]: oh yeh, typo.

Comment (vshankar) [reply doc/cephfs/client-config-ref.rst]: I meant "tells if"

File: doc/cephfs/client-config-ref.rst [hunk 2]
```diff
@@ -1,6 +1,48 @@
...
+Updating Client Configuration
+-----------------------------
+
+Certain client configurations can be applied at runtime. To check if a configuration option can be applied (taken into affect by a client) at runtime, use the `config help` command:
+
+::
+
+   ceph config help <option>
+
+   e.g.,
```
Comment (rishabh-d-dave) [inline doc/cephfs/client-config-ref.rst]: It would be nice if we only have commands inside literal block.

File: doc/cephfs/client-config-ref.rst [hunk 3]
```diff
@@ -1,6 +1,48 @@
...
-========================
+Client Configuration
+====================
+
+Updating Client Configuration
+-----------------------------
+
+Certain client configurations can be applied at runtime. To check if a configuration option can be applied (taken into affect by a client) at runtime, use the `config help` command:
+
+::
```
Comment (rishabh-d-dave) [inline doc/cephfs/client-config-ref.rst]: Just a suggestion, you can do just `command::` instead of `command:\n\n::` to get the same effect.

Comment (vshankar) [reply doc/cephfs/client-config-ref.rst]: nice!

File: doc/cephfs/client-config-ref.rst (hunk 2 above)
Comment (vshankar) [reply doc/cephfs/client-config-ref.rst]: done

**Comment by vshankar** [comment]
jenkins render docs

**Pull request merged**